build/common.sh

```diff
@@ -576,20 +545,39 @@ function kube::build::ensure_data_container() {
     local -ra docker_cmd=(
       "${DOCKER[@]}" run
       --volume "${REMOTE_OUTPUT_GOPATH}"
+      --volume /usr/local/go/pkg/linux_386_cgo
```

Comment:
An explanation or a comment would be good to have here.
Also, can it be automated?